src/registry.rs

```diff
@@ -537,11 +538,11 @@ impl NpmRegistryApi for TestNpmRegistryApi {
     name: &str,
   ) -> Result<Arc<NpmPackageInfo>, NpmRegistryPackageInfoLoadError> {
     let infos = self.package_infos.borrow();
-    Ok(Arc::new(infos.get(name).cloned().ok_or_else(|| {
```

Comment:
This clone was very noisy in benchmarks. It was causing a benchmark to run in 115ms instead of 10ms.